<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="autoScrollCheckBox">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>autoScroll</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reuseTabCheckBox">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>reuseTab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="scopesCheckBox">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>noScope</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterFileCheckBox">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="presetComboBox">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="levelSpinBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string extracomment="controls debug level of connected "/>
        </property>
        <property name="whatsThis">
         <string extracomment="controls debug level of connected "/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="qSearchLineEdit">
        <property name="whatsThis">
         <string extracomment="quick search"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="qSearchComboBox">
        <property name="toolTip">
         <string extracomment="hello3"/>
        </property>
        <property name="statusTip">
         <string extracomment="hello2"/>
        </property>
        <property name="whatsThis">
         <string extracomment="hello1"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabTrace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabFilters">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="filterFile">
        <attribute name="title">
         <string>filterFile</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTreeView" name="treeViewFile"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="filterTID">
        <attribute name="title">
         <string>filterTID</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QListView" name="listViewTID"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="filterFunc">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="title">
         <string>filterFunc</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTreeView" name="treeViewFunc"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import Log</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Log</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>Text...</string>
   </property>
  </action>
  <action name="actionFind_Prev">
   <property name="text">
    <string>Find Prev</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
